external/js/cothority/lib/misc/misc.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/** @module misc */
+
+exports.uint8ArrayToHex = uint8ArrayToHex;
+exports.hexToUint8Array = hexToUint8Array;
+exports.reverseHex = reverseHex;
+
+/**
+ * Convert a byte buffer to a hexadecimal string.
+ * @param {Uint8Array} buffer
+ *
+ * @throws {TypeError} when buffer is not Uint8Array
+ * @returns {string} hexadecimal representation
+ */
+function uint8ArrayToHex(buffer) {
```

Comment:
I don't see where these are used. Why are they in this library? If they are used for testing, can they be in the testing code? Also: error handling for invalid hex is not done.

Reply:
Yep I've added them a little bit sooner than expected. I'm working on the next PR and I'm using them only now. Will remove them in this PR.